tests/Pest.php

```diff
@@ -11,6 +11,7 @@
 uses(TestCase::class)->in(__DIR__);
 uses()->afterEach(fn () => clearGeneratedLogFiles())->in('Feature', 'Unit');
 uses()->beforeEach(fn () => Artisan::call('log-viewer:publish'))->in('Feature');
+uses()->beforeEach(fn () => Artisan::call('config:cache'))->in('Feature');
```

Comment:
Really not sure about that. Why is it needed? Caching the config (or views, routes, etc) should only be done in production for the sake of performance improvement.

Reply:
Removing this line should still make the tests pass, unless you have locally cached the config (which you shouldn't).

Reply:
@arukompas If remove this line, I can't pass my rule while it didn't receive new config

Reply:
@hungthai1401 that is not the case. Not sure how `config:cache` helps here, but one of the tests might not be passing because calling `reloadRoutes` does not remove previously-loaded routes.

For example, when the test is first set up, the `LogViewerServiceProvider::registerRoutes()` is called already with whatever config it is at that point. If `api_only` is false, then it will register the web Log Viewer routes as well.

But then when you call `reloadRoutes` test helper later on, it calls `LogViewerServiceProvider::registerRoutes()` again, and at this point `api_only` is true, so it should skip registering the web routes for Log Viewer, right? And it skips it, but it makes no difference because the web routes have been registered already when setting up the test initially.

Need to find a way to update the `reloadRoutes` method so that it first **unloads** (unsets) previously set Log Viewer routes before calling `LogViewerServiceProvider::registerRoutes()` again.

Reply:
You can also see that `config:cache` doesn't really help when running the full test suite, as shown in the GitHub actions run on this PR.

Reply:
@arukompas I have update `Pest.php` when loading base test case to ignore my test case. Plz check my latest commit. Thanks